### DISCUSSION TITLE
please, change all "private" to "protected" class vars

### DISCUSSION THREAD

**Discussion by someson** [phalcon/cphalcon#15958] [General]
subj. now with all those "private"s doesn't let ones extend the classes. The most of "private"s don't even have "get" methods!

**Comment by someson** [comment]
affects both phalcon4 and phalcon5, e.g. 
```php
class NativeArray extends AbstractAdapter implements ArrayAccess
{
    /**
     * @var array
     */
    private translate;

...
```
really?

**Comment by Jeckerson** [comment]
Hello, I think it is better to create an issue instead of topic in discussion.

**Reply by someson** [reply]
for some people it isn't an issue, till they try to extend some of those. I thought maybe there's a reason why the implementation contains such a "solution" and sure wanna hear it, that's why discussion is more appropriate, i think. But sure i could create an issue in addition...
